git-coco

```diff
@@ -184,7 +256,10 @@ case "$command" in
     pr)
         pr "$@" || exit 1
         ;;
+    prlist)
+        prlist "$@" || exit 1
+        ;;
     *)
-        fail "Usage: git coco ls|create|clone|rm|init|pr"
+        fail "[-] Usage: git coco ls|create|clone|rm|init|pr|prlist"
```

Comment:
It's just my opinion, but I don't like the `[-]` part because it's not a standard part of other tools -- when I see it I have to figure out why it's there when no other tool has it.